app/templates/suppliers/edit_registered_address.html

```diff
@@ -28,11 +28,8 @@
 {% endblock %}
 
 {% block main_content %}
-
   {% include 'toolkit/forms/validation.html' %}
-  {% with heading = "Update registered company address for ‘{}’".format(supplier.name) %}
-    {% include "toolkit/page-heading.html" %}
-  {% endwith %}
+  <h1 class="govuk-heading-xl">{{ "Update registered company address for ‘{}’".format(supplier.name) }}</h1>
```

Comment:
This too can just be `<h1 class="govuk-heading-xl">Update registered company address for ‘{{ supplier.name }}’</h1>`